### DISCUSSION TITLE
How to work with other Bevy versions

### DISCUSSION THREAD

**Discussion by bit-garden** [jcornaz/heron#279] [Q&A] [answered]
The current releases for Bevy are broken for Android. I have a local version with a patch that let's it work, but I am getting errors(ex: `PhysicsPlugin` does not implement `Plugin`) which seems to be caused by mixed versions of Bevy.

How would I be able to work with Heron with my patched Bevy?

**Comment by jcornaz** [comment]
You have to pick a version of heron that is compatible with the bevy version you use.

The table describing which version of heron is compatible with which version of bevy is documented in the [readme](https://github.com/jcornaz/heron#supported-bevy-versions).

**Comment by jcornaz** [comment, marked as answer]
> How would I be able to work with Heron with my patched Bevy?

Unreleased versions of bevy are not supported. If you have problems using bevy "main" or a fork of bevy, the solution is to fork heron and update that fork to point to whatever fork/branch of bevy you need.

**Reply by bit-garden** [reply]
Is it just as simple as updating the cargo.toml files or is there something else?

**Reply by bit-garden** [reply]
Ended up answering my own question. Updating cargo.toml files worked for the most part. There is an upcoming API change to the Time api that needed to be fixed, but it ended up being mostly painless.
